### DISCUSSION TITLE
Backup of a PBS to USB drive

### DISCUSSION THREAD

**Discussion by eo62** [psy0rz/zfs_autobackup#275] [Q&A]
Is it possible to use zfs_autobackup to backup a PBS server pool to an USB drive?

**Comment by psy0rz** [comment]
i dont know much about PBX, but you can use zfs-autobackup for local backups with no problem

**Reply by psy0rz** [reply]
(just create a pool on the USB drive, and when you reconnect the USB drive you might need to zpool import it)
